README.md

```diff
@@ -52,10 +51,19 @@ documentation][docs]
 
 [examples]: ./examples
 
+## Options
+
+- `onError` (function, default is undefined). Used to ensure that the function
+  does not throw. **Note:** When applied, the function will return `204`
+  regardless.
+
+- `parseJson` (boolean, default is `true`). When set to true, uses `JSON.parse`
```

Comment:
Good enough for now, but might consider supporting a `parser` option, that takes the raw message and returns T. So you could use runtypes are whatever before it touches your handler. Not sure if overkill

Reply:
Added a separate issue on this: https://github.com/cobraz/pubsub-http-handler/issues/22